templates/definition/meter/sonnenbatterie_eco56.yaml

```diff
@@ -48,5 +50,58 @@ render: |
     source: http
     uri: http://{{ .host }}:{{ .port }}/rest/devices/battery
     jq: .M30 # SOC relative to usable capacity (.M05 # display SOC)
+  batterymode:
+    # use a sequence to propagate resets to the watchdog
+    source: sequence
+    set:
+      - source: switch
+        switch:
+          - case: 1 # normal
+            set:
+              source: http
+              method: PUT
+              uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+              body: '10' # Automatic
+          - case: 2 # hold
+            set:
+              source: http
+              method: PUT
+              uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+              body: '20' # Standby
+          - case: 3 # charge
+            set:
+              source: http
+              method: PUT
+              uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+              body: '55' # slave mode
+      - source: watchdog
+        timeout: 30s # 3 minutes without setting a value will stop all charging
+        reset: 1,2
```

Comment:
Hast Du das getestet? Reset kann laut Code nur einen Wert.

Reply:
Deshalb habe ich das in c34d8df6b85582a3f11ee180972bbcfe4b5a2f99 eingebaut

Reply:
Ah, nice. Einfacher: mach config.Reset `[]string`, dann sparst Du das Splitting. Leere Slices einfach als

    var reset []int64